linux/preview/examples/mssql-mlservices/run-container

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+docker container run \
+    --name sql-ml \
+    --env 'ACCEPT_EULA=Y' \
+    --env 'MSSQL_PID=Developer' \
+    --env 'ACCEPT_EULA_ML=Y' \
+    --env 'SA_PASSWORD=My!SuperSecretPassw0rd' \
```

Comment:
are both SA_PASSWORD and MSSQL_SA_PASSWORD needed ? From the Readme I thought that MSSQL_SA_PASSWORD replaced SA_PASSWORD

Reply:
You are right - SA_PASSWORD is not needed.

Reply:
Correct, just leave the MSSQL_SA_PASSWORD env variable.